contrib/jmx-metrics/src/main/java/io/opentelemetry/contrib/jmxmetrics/ClientCallbackHandler.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.contrib.jmxmetrics;
+
+import javax.annotation.Nullable;
+import javax.security.auth.callback.Callback;
+import javax.security.auth.callback.CallbackHandler;
+import javax.security.auth.callback.NameCallback;
+import javax.security.auth.callback.PasswordCallback;
+import javax.security.auth.callback.UnsupportedCallbackException;
+import javax.security.sasl.RealmCallback;
+
+public class ClientCallbackHandler implements CallbackHandler {
```

Comment:
Instead of adding our own options for authentication, can we just let users use the standard JMX pattern like `com.sun.management.jmxremote.password.file`? Less surface for security that we have to think about, the better, even if the current implementation is sane.

Reply:
I'm not certain, but I believe that file is only for setting up the mbean server, similar to the access file, and establishing a client connection requires the connector to use the `jmx.remote.credentials` (not a property?) string array entry.

Reply:
https://docs.oracle.com/cd/E19698-01/816-7609/6mdjrf86t/index.html